components/suggestion/command_provider/app_command_parser/app_command_parser.ts

```diff
@@ -595,19 +595,21 @@ export class AppCommandParser {
         const result: AutocompleteSuggestion[] = [];
 
         const bindings = this.getCommandBindings();
+
         for (const binding of bindings) {
-            let base = binding.app_id;
+            let base = binding.label;
             if (!base) {
                 continue;
             }
 
             if (base[0] !== '/') {
                 base = '/' + base;
             }
+
             if (base.startsWith(command)) {
                 result.push({
+                    Complete: binding.label,
                     Suggestion: base,
-                    Complete: base.substring(1),
```

Comment:
Do we want to support the scenario where an app has put a slash in front of the label? This code will no longer support this because of this change ☝️

Reply:
@mickmister I'm would like to keep the client "dump". Any special case handling should go to the server i.e. Apps plugin as it's easier to change things there.

I would like to tackle handling that case as part of a larger effort to add an `IsValid()` method to bindings.